galactic.ignore

```diff
@@ -0,0 +1 @@
+test_osrf_testing_tools_cpp
```

Comment:
Not sure if it makes sense to include this. The next distro will be `rolling` and I would expect that `galactic` will get it's configuration copied over.

Reply:
I am sure the scripts for the `rolling` distro will take care of that - otherwise you would end up with various cases which need manual intervention. For @nuclearsandwich to clarify if that is actually the case.

Reply:
@nuclearsandwich Friendly ping.

Reply:
Actually.. this isn't something that I explicitly handle. Despite there being multiple packages in my test distributions which have ignore files none of them ever caused problems. I was just finishing the last test run before throwing the big switch. So I'll open a PR covering ignored packages before I do.

Reply:
https://github.com/ros/rosdistro/pull/25477